docs/deploy/index.md

```diff
@@ -930,6 +930,11 @@ When requesting the execution of a specific playbook, the script will force exec
 
 In case of a failure, the install script can be reapplied as most of the settings are idempotent. The script contains a checkpointing mechanism, each successful target will have a `.ok` file created in the playbooks directory. If you want to re-apply a target, delete this file and rerun the install script.
 
+Tip: you can pass `--tags` in order to re-run just a specific role or task in an ansible playbook, e.g.
+```bash
+./install.sh ood --tags paraview
```

Comment:
Please change paraview to something else as we don't have paraview tag. i would use --tags foo

Reply:
there is a paraview tag
https://github.com/Azure/az-hop/blob/0b4b3ee0692481416b9438dd7df131157b532340/playbooks/roles/ood-applications/tasks/main.yml#L25

but foo is good with me :-)

Reply:
:-) was not aware of that tag. what install will do if only the paraview tag is provided ? will it only execute tasks tagged with paraview ? if so you can leave it.